README.md

```diff
@@ -17,6 +17,7 @@ Quick start:
 1. Click [Use this template](https://github.com/keptn-sandbox/keptn-service-template-go/generate) on top of the repository, or download the repo as a zip-file, extract it into a new folder named after the service you want to create (e.g., simple-service) 
 1. Replace every occurrence of (docker) image names and tags from `keptnsandbox/keptn-service-template-go` to your docker organization and image name (e.g., `yourorganization/simple-service`)
 1. Replace every occurrence of `keptn-service-template-go` with the name of your service (e.g., `simple-service`)
+1. Replace every occurrence of `keptn-service` with the name of your service (e.g., `simple-service`)
```

Comment:
should this be `keptnservice` (without the `-`?)

Reply:
Btw this line is gonna disappear while fixing the [next comment ](https://github.com/keptn-sandbox/keptn-service-template-go/pull/92#discussion_r788466424) since that was the only reference left ;)